mkosi/__init__.py

```diff
@@ -1814,6 +1834,12 @@ def install_type1(
 
     kmods = build_kernel_modules_initrd(context, kver)
 
+    dtb = None
+    if context.config.devicetree:
+        dtb = dst / context.config.devicetree
+        with umask(~0o700):
```

Comment:
Is there a reason for this strict umask? Can a devicetree file be sensitive or is this required by whatever is loading the devicetree file?

Reply:
I picked the same umask as for dst (the parent) and systemd-boot loader entries directories. There's nothing sensitive and all these things typically ends up in a vfat partition.